### PR TITLE
luminous: ceph-volume: allow using a device or partition for `lvm --data`

### PR DESCRIPTION
**Pull request by andrewschoen**
backport of #18924

### CODE REVIEW

File: doc/ceph-volume/lvm/prepare.rst [hunk 1]
```diff
@@ -33,17 +33,37 @@ the back end can be specified with:
...
+volumes other than following the minimum size requirements for data and
+journal.
 
 The API call looks like::
 
     ceph-volume prepare --filestore --data data --journal journal
 
-The journal *must* be a logical volume, just like the data volume, and that
-argument is always required even if both live under the same group.
+There is flexibility to use a raw device or partition as well for ``--data``
```
Comment (amitkumar50) [inline doc/ceph-volume/lvm/prepare.rst]: raw device or partitioned?

File: doc/ceph-volume/lvm/prepare.rst [hunk 2]
```diff
@@ -33,17 +33,37 @@ the back end can be specified with:
...
+that will be converted to a logical volume. This is not ideal in all situations
```
Comment (amitkumar50) [inline doc/ceph-volume/lvm/prepare.rst]: is not ideal for all the situations

**Comment by alfredodeza** [comment]
jenkins test ceph-volume tox

**Comment by andrewschoen** [comment]
jenkins test ceph-volume tox

**Pull request merged**